pkgs/shells/zsh/agdsn-zsh-config/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, zsh
+, coreutils
+, inetutils
+, procps
+}:
+
+with lib;
+
+stdenv.mkDerivation rec {
+  pname = "agdsn-zsh-config";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "agdsn";
+    repo = "agdsn-zsh-config";
+    rev = "v${version}";
+    sha256 = "sha256-/l2fE4ZsZ6f89fYG9sTEV1mrXZ3MLXx6K3CTUQHiAsc=";
+  };
+
+  dontBuild = true;
+
+  strictDeps = true;
+  buildInputs = [ zsh coreutils procps ]
+    ++ optional stdenv.isLinux inetutils;
```

Comment:
Why are these buildInputs added?

Reply:
I carried those over from `grml-zsh-config`. For my personal use case (which is calling this package's zshrc from `programs.zsh.interactiveShellInit`) everything works fine without these buildInputs. However, I assume there was a reason at some point for including them, and I didn't wanna risk breaking someone else's usecase who might be switching over from grml.

Reply:
`grml-zsh-config` have a buildPhase that supposedly uses these buildInputs, while this package does not. You can check the build outputs with or without them to seee if there's any difference.

Reply:
I don't see any difference in the build outputs with or without these buildInputs, so I'm removing them. Thank you for the hint.